### PR TITLE
Get rid of all warnings from github

### PR DESCRIPTION
**Pull request by SiegfriedSobkowiak**
# Update github actions
`update actions`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one E2E Test exists testing the new feature
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [x] The new version is deployed to a dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [ ] All implemented Social Logins have been tested manually in the UI
- [ ] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [ ] The local Dev stack still works: execute `startDevelopmentStack.sh`, execute npm run cypress and run Cypress Tests locally

### CODE REVIEW

File: .github/workflows/CypressImage.yaml [hunk 1]
```diff
@@ -37,14 +37,14 @@ jobs:
           cp -r /home/runner/.gradle ./gradlecache
           cp -r /home/runner/.cache/Cypress ./cypresscache
       - name: Login to Github Packages
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
```
Comment (ppuetsch) [inline .github/workflows/CypressImage.yaml:40]: Soweit ich das sehen konnte gibt es login-action nur in v2?!
Aber im Code gibt es noch eine Stelle wo v1 benutzt wird.

File: .github/workflows/CypressImage.yaml [hunk 2]
```diff
@@ -37,14 +37,14 @@ jobs:
...
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Set up Docker Buildx
         id: buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
```
Comment (ppuetsch) [inline .github/workflows/CypressImage.yaml:47]: Soweit ich das sehen konnte gibt es setup-buildx-aciton nur in v2?!

Comment (SiegfriedSobkowiak) [reply .github/workflows/CypressImage.yaml]: stimmt und es ist in der CI nicht aufgefallen weil die beiden container nicht Teil der CI sind

File: .github/workflows/CypressImage.yaml (hunk 1 above)
Comment (SiegfriedSobkowiak) [reply .github/workflows/CypressImage.yaml]: beides angepasst

**Pull request merged**